serverless/src/models/vo/responseVO.ts

```diff
@@ -0,0 +1,4 @@
+export class ResponseVO {
+    statusCode: number;
+    body: string;
+}
```

Comment:
VO?

Reply:
I'll be renaming this to just be Response, VO (Virtual Object) is more of a best practice thing for Java servers.